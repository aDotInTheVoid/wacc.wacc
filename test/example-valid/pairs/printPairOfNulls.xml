<unit>
  <main>
    <addr_local>0</addr_local>
    <null/>
    <null/>
    <newpair/>
    <assign_do/>
    <local>0</local>
    <print><kind>ptr</kind><newline>false</newline></print>
    <str>" = ("</str>
    <print><kind>str</kind><newline>false</newline></print>
    <addr_local>1</addr_local>
    <local>0</local>
    <fst/>
    <assign_do/>
    <local>1</local>
    <print><kind>ptr</kind><newline>false</newline></print>
    <str>","</str>
    <print><kind>str</kind><newline>false</newline></print>
    <addr_local>2</addr_local>
    <local>0</local>
    <snd/>
    <assign_do/>
    <local>2</local>
    <print><kind>ptr</kind><newline>false</newline></print>
    <str>")"</str>
    <print><kind>str</kind><newline>true</newline></print>
  </main>
</unit>