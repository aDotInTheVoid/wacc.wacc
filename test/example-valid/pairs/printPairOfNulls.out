0x{{PTR}} = ((nil),(nil))
